projects/demo/src/modules/experimental/card/examples/5/index.less

```diff
@@ -0,0 +1,82 @@
+:host {
+    display: flex;
+    gap: 1.2rem;
+    min-width: 40.625rem;
+}
+
+.badge {
+    background: #aff218;
+    color: #333;
+}
+
+.money {
+    background: #ffdd2d;
+    color: #333;
+}
+
+.poster[data-surface] {
+    background: rgb(88, 192, 190);
+    padding: 0.75rem;
+    color: #fff;
+    width: 13.9375rem;
+    height: 8rem;
+
+    [tuiSubtitle] {
+        color: #fff;
+    }
+
+    &:after {
+        background: url(/assets/images/poster.svg) bottom right no-repeat;
+    }
+
+    &:hover:after {
+        transform: scale(1.1);
+    }
+}
+
+.fly[data-surface] {
+    background: rgb(101, 174, 234);
+    padding: 0.75rem;
+    color: #fff;
+    width: 8rem;
+    height: 8rem;
+
+    &:after {
+        background: url(/assets/images/fly.svg) bottom right no-repeat;
+    }
+
+    &:hover:after {
+        transform: scale(1.1);
+    }
+}
+
+.google[data-surface],
+.microsoft[data-surface] {
+    padding: 0.75rem;
+    background: var(--tui-base-02);
+    height: 8rem;
+
+    &:hover:after {
+        transform: scale(1.1);
+    }
+}
+
+.google[data-surface] {
```

Comment:
Per my other comment about specificity, I believe once size variable is added, we can remove those attributes from everywhere. Please give it a shot.

Reply:
fixed